src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +513,78 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if get_nearest_transcript_junction:
```

Comment:
This is a style preference. Don't be afraid to add new lines in between sections. It makes it easier to read IMO